appeals/web/src/server/appeals/appeal-details/inspector-access/inspector-access.mapper.js

```diff
@@ -0,0 +1,76 @@
+/**
+ * @typedef {import('../appeal-details.types.js').WebAppeal} Appeal
+ */
+
+import { appealShortReference } from '#lib/appeals-formatter.js';
+import { errorInspectorAccessRadio } from '#lib/error-handlers/change-screen-error-handlers.js';
+import { conditionalFormatter } from '#lib/mappers/global-mapper-formatter.js';
+
+/**
+ * @param {Appeal} appealData
+ * @param {{radio: string, details: string}} storedSessionData
+ * @param {string} origin
+ * @param {string} source
+ * @param {import("@pins/express").ValidationErrors | undefined} errors
+ * @returns {PageContent}
+ */
+export const changeInspectorAccessPage = (
+	appealData,
+	storedSessionData,
+	origin,
```

Comment:
If this is only being used for the backlink URL, it would probably be clearer to name it `backLinkUrl` or similar